notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<p>There seems to be a missing explanation section for 'Point-in-Time Correctness'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>Agreed. We can set the stage as we need to backfill data, and then explain the link between point-in-time correctness and backfilling data...which seems to be the scenario we're describing based on the steps below, but I'm not sure if that's right?</p>